lib/index.js

```diff
@@ -65,6 +65,11 @@ HubSpot.prototype.identify = function(identify) {
   var traits = identify.traits({ firstName: 'firstname', lastName: 'lastname' });
   traits = convertDates(traits);
   traits = formatTraits(traits);
+
+  if (identify.companyName() !== undefined) {
```

Comment:
might be better to set this before `formatTraits` though actually in this case i dont think it matters

Reply:
Indeed, it doesn't. Are we going to be making similar changes in the future? That could justify choosing one way or another.

Reply:
Also tbh I put it there because I was certain that `formatTraits` returns a plain-old-JS-object, and I wasn't sure about `identify.traits`